'logic_error': is not a member of 'std'

Hello. On v1.5.1 I get some compiler errors from MSVC:
```
<...>\HighsHashTree.h(340,22): error C2039: 'logic_error': is not a member of 'std'
```
Should HighsHashTree.h have `#include <stdexcept>`?

--- a/src/util/HighsHashTree.h
+++ b/src/util/HighsHashTree.h
@@ -11,6 +11,8 @@
 #ifndef HIGHS_UTIL_HASH_TREE_H_
 #define HIGHS_UTIL_HASH_TREE_H_
 
+#include <stdexcept>
+
 #include "util/HighsHash.h"
 
 using std::memcpy;

